### ISSUE TITLE
Stringifying `content`

### ISSUE THREAD

**Issue by SntsDev** [storyblok/gatsby-source-storyblok#9]
The `content` field is stringified. This is good for getting all the content for different schemas, but it has a problem, that of not being able to use graphql to use other functions. For instance, use `gatsby-plugin-remote-images`. Is there a possibility of having a parameter to enable/disable this?

https://github.com/storyblok/gatsby-source-storyblok/blob/327b3aae79200d0ca17c5a2cd77498c7f259d599/gatsby-node.js#L112

**Comment by onefriendaday** [comment]
I assume you want to use the functionalities of gatsby-image to optimize Storyblok images, right? You can do that with a simple React helper that transforms the images with Storyblok's image optimizing service: https://www.storyblok.com/docs/image-service#code-examples

**Comment by SntsDev** [comment]
Thanks.
I don't really need that exactly. `gatsby-image` (<Img .../>) does more of things apart from resizing or cropping. For instance, I'm interested on using it to show a low-res thumbnail and show, using the intersectionObserver and the screen characteristics, an optimized version with lazy-loading. Therefore, I need to download the images. But the fact that content is stringified makes it difficult to add new nodes with the local url.

It would be a good solution to expose the content object both as a string and an object. For instance, to keep backwards compatibility keep the name `content` for the stringified version and `contentObject` for the non-stringified version:
```javascript
{
    content: "\{\"title\":\"Hello World\"\}",
    contentObject: { title: "Hello World" }
}
```
In that way, your examples will continue to work and if we need that we can use graphql on the other field.

**Comment by onefriendaday** [comment]
Its now available in version 0.2.0:
https://github.com/storyblok/gatsby-source-storyblok#filtering-on-content-type-fields

You can access every content type field with "field_YOUR_FIELD_NAME".

**Comment by chrish-d** [comment]
I can't see how this allows you to access images within the `content` field? My image field isn't a content type field so the update above doesn't expose this. Any suggestions? 😄 @onefriendaday
